network/forks/fork.go

```diff
@@ -182,3 +182,21 @@ func SortedForkVersions(forkSchedule map[[4]byte]primitives.Epoch) [][4]byte {
 	})
 	return sortedVersions
 }
+
+// LastForkEpoch returns the last valid fork epoch that exists in our
+// fork schedule.
+func LastForkEpoch() primitives.Epoch {
+	fSchedule := params.BeaconConfig().ForkVersionSchedule
+	sortedForkVersions := SortedForkVersions(fSchedule)
+	lastValidEpoch := primitives.Epoch(0)
+	for _, v := range sortedForkVersions {
```

Comment:
Another way of doing this would be to loop in reverse and return the first non-max epoch